assets/js/modules/analytics/components/common/AccountCreate/CountrySelect.js

```diff
@@ -48,7 +48,7 @@ export default function CountrySelect() {
 				timezone: countriesByCode[ newCountryCode ].defaultTimeZoneId,
 			} );
 		}
-	}, [ setValues ] );
+	}, [ setValues, value ] );
```

Comment:
I don't know how we haven't got bugs from some of these! As here this would have had a stale `value`